onnx/backend/test/runner/__init__.py

```diff
@@ -256,6 +260,7 @@ def _add_test(
         test_func: Callable[..., Any],
         report_item: list[ModelProto | NodeProto | None],
         devices: Iterable[str] = ("CPU", "CUDA"),
+        **kwargs1: Any,
```

Comment:
```suggestion
        **test_kwargs: Any,
```

avoid numbers because they are not meaningful